risedev.yml

```diff
@@ -538,6 +538,9 @@ template:
     # 7 days by default.
     min-sst-retention-time-sec: 604800
 
+    # Enable sanity check when SSTs are committed. Disabled ny default.
```

Comment:
```suggestion
    # Enable sanity check when SSTs are committed. Disabled by default.
```

Reply:
How about enable this option in some CI workflows?

Reply:
Yes, I've enabled it for some risedev config.
The option doesn't affect overall CI execution time.